src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,37 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const currentTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  const handleClick = (tab) => {
+    if (tab.id !== selectedTabId) {
+      onTabSelected(tab);
+    }
+  };
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              className={tab.id === selectedTabId && 'is-active'}
```

Comment:
```suggestion
              className={tab.id === currentTab.id && 'is-active'}
```

i think it should fix tests